src/apify_client/clients/resource_clients/webhook_collection.py

```diff
@@ -0,0 +1,70 @@
+from typing import Any, Dict, List, Optional
+
+from ..base.resource_collection_client import ResourceCollectionClient
+from .webhook import prepare_webhook_representation
+
+
+class WebhookCollectionClient(ResourceCollectionClient):
+    """Sub-client for manipulating webhooks."""
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        """Initialize the WebhookCollectionClient."""
+        super().__init__(*args, resource_path='webhooks', **kwargs)
+
+    def list(self, *, limit: Optional[int] = None, offset: Optional[int] = None, desc: Optional[bool] = None) -> Dict:
+        """List the available webhooks.
+
+        https://docs.apify.com/api/v2#/reference/webhooks/webhook-collection/get-list-of-webhooks
+
+        Args:
+            limit (int, optional): How many webhooks to retrieve
+            offset (int, optional): What webhook to include as first when retrieving the list
+            desc (bool, optional): Whether to sort the webhooks in descending order based on their date of creation
+
+        Returns:
+            The list of available webhooks matching the specified filters.
+        """
+        return self._list(limit=limit, offset=offset, desc=desc)
+
+    def create(
+        self,
+        event_types: List,
+        request_url: str,
+        payload_template: Optional[str] = None,
+        actor_id: Optional[str] = None,
+        actor_task_id: Optional[str] = None,
+        actor_run_id: Optional[str] = None,
+        ignore_ssl_errors: Optional[bool] = None,
+        do_not_retry: Optional[bool] = None,
+        idempotency_key: Optional[str] = None,
+        is_ad_hoc: Optional[bool] = None,
+    ) -> Dict:
+        """Create a new webhook.
+
+        You have to specify exactly one out of actor_id, actor_task_id or actor_run_id.
```

Comment:
If they don't API errors out?

Reply:
Haven't tested it, but am 99% sure that if you don't include any, it raises an error, whereas more than one will get saved no problem. However,  specifying more than one has no function and might lead to unexpected behavior.